### PR TITLE
Remove deprecated components

### PR DESCRIPTION
**Pull request by James300**
- Removed Red/Green/Outline Button
- Replaced Banner iconName with icon

### CODE REVIEW

**Comment by James300** [comment]
[Remove Deprecated Components](https://trello.com/c/LwaJq137/37-remove-deprecated-components)

File: MIGRATING_to_v3.md [hunk 1]
```diff
@@ -31,6 +34,18 @@ import { ThumbsUp } from 'pcln-icons
...
+
+```
+import { Banner } from 'pcln-design-system'
+import { ThumbsUp } from 'pcln-icons'
+
+<Banner icon={<ThumbsUp />} />
+```
+
 - Hug
   - Icon prop now takes a `node` instead of an icon name
```
Comment (sdalonzo) [inline MIGRATING_to_v3.md]: maybe match this to the Banner msg you added above?

File: MIGRATING_to_v3.md [hunk 2]
```diff
@@ -17,6 +17,9 @@
 ### Core (`pcln-design-system`)
 
 - peerDependency increase: `pcln-icons >=3`
+- GreenButton `<GreenButton />` has been removed
+- RedButton `<RedButton />` has been removed
+- OutlineButton `<OutlineButton />` has been removed
```
Comment (sdalonzo) [inline MIGRATING_to_v3.md]: are there examples of how to use the variation instead in the storybook?

Comment (sdalonzo) [reply MIGRATING_to_v3.md]: we should mention variations in the migration guide

Comment (James300) [reply MIGRATING_to_v3.md]: There are examples in the storybook but I'll add the instructions to the migration guide

**Pull request merged**